blazor/datagrid/row.md

```diff
@@ -9,146 +9,1051 @@ documentation: ug
+            this.IsVerified = IsVerified;
+            this.ShipCity = ShipCity;
+            this.ShipCountry = ShipCountry;
+        }
+
+        public static List<OrderData> GetAllRecords()
         {
-            args.Row.AddClass(new string[] { "above-35" });
+            if (Orders.Count == 0)
+            {
+                Orders.Add(new OrderData(10248, "VINET", "Vins et alcools Chevalier", 32.38, new DateTime(1996, 7, 4), new DateTime(1996, 08, 07), true, "Reims", "France"));
+                Orders.Add(new OrderData(10249, "TOMSP", "Toms Spezialitäten", 11.61, new DateTime(1996, 7, 5), new DateTime(1996, 08, 07), false, "Münster", "Germany"));
+                Orders.Add(new OrderData(10250, "HANAR", "Hanari Carnes", 65.83, new DateTime(1996, 7, 6), new DateTime(1996, 08, 07), true, "Rio de Janeiro", "Brazil"));
+                Orders.Add(new OrderData(10251, "VINET", "Vins et alcools Chevalier", 41.34, new DateTime(1996, 7, 7), new DateTime(1996, 08, 07), false, "Lyon", "France"));
+                Orders.Add(new OrderData(10252, "SUPRD", "Suprêmes délices", 151.30, new DateTime(1996, 7, 8), new DateTime(1996, 08, 07), true, "Charleroi", "Belgium"));
+                Orders.Add(new OrderData(10253, "HANAR", "Hanari Carnes", 58.17, new DateTime(1996, 7, 9), new DateTime(1996, 08, 07), false, "Bern", "Switzerland"));
+                Orders.Add(new OrderData(10254, "CHOPS", "Chop-suey Chinese", 22.98, new DateTime(1996, 7, 10), new DateTime(1996, 08, 07), true, "Genève", "Switzerland"));
+                Orders.Add(new OrderData(10255, "VINET", "Vins et alcools Chevalier", 148.33, new DateTime(1996, 7, 11), new DateTime(1996, 08, 07), false, "Resende", "Brazil"));
+                Orders.Add(new OrderData(10256, "HANAR", "Hanari Carnes", 13.97, new DateTime(1996, 7, 12), new DateTime(1996, 08, 07), true, "Paris", "France"));
+            }
+            return Orders;
         }
+
+        public int? OrderID { get; set; }
+        public string CustomerID { get; set; }
+        public string ShipName { get; set; } 
+        public double? Freight { get; set; }
+        public DateTime? OrderDate { get; set; }
+        public DateTime? ShippedDate { get; set; }
+        public bool? IsVerified { get; set; }
+        public string ShipCity { get; set; }
+        public string ShipCountry { get; set; }
     }
 }
-```
+{% endhighlight %}
+{% endtabs %}
 
-![Customizing Blazor DataGrid Rows](./images/blazor-datagrid-rows-customization.png)
+{% previewsample "https://blazorplayground.syncfusion.com/embed/hjrTMjUtTMnyFhpX?appbar=false&editor=false&result=true&errorlist=false&theme=bootstrap5" %}
 
-### Using CSS customize alternate rows
+> Frozen rows should not be set outside the grid view port.
+> Frozen Grid will support row virtualization feature, which helps to improve the Grid performance while loading a large dataset.
+> The frozen feature is supported only for the rows that are visible in the current view.
```

Comment:
what this sentence means?

Reply:
Frozen feature supports only for the visible row in the current view page if we give more frozenrows value than current view page it will not work as expected. That's why that point is added mam
![image](https://github.com/user-attachments/assets/b2da475f-efc4-42ec-a83f-947f365707c9)
![image](https://github.com/user-attachments/assets/b71f5c81-98c5-4bf5-b5c3-9c06aa77d2b4)

Sample: https://blazorplayground.syncfusion.com/rXBfMsrlLxqJMHsI